themes/10up-theme/includes/blocks.php

```diff
@@ -113,6 +113,16 @@ function blocks_editor_styles() {
 		Utility\get_asset_info( 'editor-style-overrides', 'version' )
 	);
 
+	if ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) {
```

Comment:
I could also see using `wp_get_environment_type( 'local' )` here.

Reply:
That could work too, but HMR only works when SCRIPT_DEBUG is set to true and this enqueue only exists for the purpose of HMR ... so even if the env is set to local the SCRIPT_DEBUG constant could still be false.